app/models/repository.rb

```diff
@@ -1,13 +1,35 @@
 class Repository
-  attr_reader :all, :name, :url, :description, :language, :description, :language, :last_updated
-  def initialize(attrs)
+  attr_reader :all, :name, :url, :description, :description,
+              :language, :last_updated, :stars, :stars_url, :forks, :starred, :creator, :forked
+
+  def initialize(attrs, starred = false, forked = false)
     @name = attrs[:name]
     @url = attrs[:html_url]
     @description = attrs[:description]
```

Comment:
@icorson3 @jmejia I also didn't figure out how to make this attribute have either the actual description or a blank string. Some things I tried included attrs[:description].to_s and breaking description into its own method, then checking if it was nil and then trying to convert the attribute key. But I still couldn't get it. Is that the right concept? Any additional hints?

Reply:
Actually nevermind, I just needed a walk, it was a syntax error.

Reply:
👍